packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -38,25 +49,41 @@ export default function Onboarding() {
     },
   ];
 
-  const title: string = t('title');
-  const description: string = t('description');
   const imgSrc = {
     src: onboardingImgSrc,
   };
 
   return (
-    <OnboardingLayout
-      title={title}
-      img={imgSrc}
-      description={description}
-      orderButtonLabel={t('orderButtonLabel')}
-      orderHref={t('orderButtonLink')}
-      moreInfoButtonLabel={t('moreInfoButtonLabel')}
-      moreInfoHref={t('moreInfoButtonLink')}
-    >
-      {tileList.map((tile) => (
-        <Card key={tile.id} href={tile.href} texts={tile.texts} />
-      ))}
-    </OnboardingLayout>
+    <>
+      <Breadcrumb />
+      <OnboardingLayout
+        title={t('title')}
+        img={imgSrc}
+        description={
+          <div className="text-center">
+            <OdsText preset={ODS_TEXT_PRESET.heading2}>{t('titlebis')}</OdsText>
+            <OdsText className="block">{t('description')}</OdsText>
+            <div>
+              <ul className="mt-2 text-sm text-center w-[10%] mx-auto">
+                <li>{t('ip_onboarding_content_list_item_1')}</li>
+                <li>{t('ip_onboarding_content_list_item_2')}</li>
+              </ul>
+            </div>
+            <OdsText className=" w-[78%] mt-4">
+              <span>{t('descriptionBis')}</span>
+              <Links href={link.presentationLink}></Links>
```

Comment:
There is no text inside the link ?

Reply:
This link component doesn't take children